server/schemas/typeDefs.js

```diff
@@ -43,7 +43,7 @@ const typeDefs = gql`
   }
   type Mutation {
     addUser(username: String!, email: String!, password: String!): Auth
-    addEvent(title: String!, address: String!, description: String!, owner: String!, capacity: Int!, when: String!): Event
+    addEvent(title: String!, address: String!, description: String!, owner: String, capacity: Int!, when: String!): Event
```

Comment:
What is owner here? Is that the same as user?

Reply:
Correct user is the same thing as owner. When sivan made the backend database models, she had the username field called owner